mmv1/templates/terraform/examples/volume_basic.tf.erb

```diff
@@ -0,0 +1,29 @@
+resource "google_netapp_storage_pool" "default" {
+  name = "<%= ctx[:vars]['pool_name'] %>"
+  location = "us-central1"
+  service_level = "PREMIUM"
+  capacity_gib = "2048"
+  network = data.google_compute_network.default.id
+}
+
+
+resource "google_netapp_volume" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
let's keep only the minimum (required) fields needed for volume creation in this basic example/test. That said - we can remove `smb_settings`, `unix_permissions`, `labels`, `description`, `snapshot_directory`, `security_style` and `kerberos_enabled`

Reply:
ack